client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/MiscellaneousDocumentationIT.java

```diff
@@ -31,20 +31,10 @@
 import java.io.IOException;
 
 /**
- * This class is used to generate the Java Main API documentation.
- * You need to wrap your code between two tags like:
- * // tag::example[]
- * // end::example[]
- *
- * Where example is your tag name.
```

Comment:
I don't think we need these comments any more. They were useful when I first built the extraction tool for education but I think something like what I have now is shorter and easier to read. Ideas? Suggestions?

Reply:
+1